firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkUpdater.java

```diff
@@ -203,7 +205,8 @@ private static boolean isResponseSuccess(int responseCode) {
     return responseCode >= 200 && responseCode < 300;
   }
 
-  private long downloadToDisk(
+  @VisibleForTesting
```

Comment:
Wasn't sure how to test this without this annotation but seemed like the best case without a big refactor.